### ISSUE TITLE
ArgumentCountOutOfRange error when command includes extra whitespace

### ISSUE THREAD

**Issue by f2bo** [dotnet/HttpRepl#233]
I notice that anything other than a single space character in a command is interpreted as one or more additional arguments. For example, the following command works as expected:
```
GET http://xxx.yyy
```
Whereas including extra space between the GET and the URL (or at the end of the line) 
```
GET  http://xxx.yyy
```
results in the following error, where the number in the error message matches the count of additional space characters.
```
ArgumentCountOutOfRange -- 2
Usage: GET [Options]

Issues a GET request.
```
I'm using 
```
microsoft.dotnet-httprepl 3.0.0-preview.19460.2
```

**Comment by granttrevor** [comment]
Currently experiencing something along these lines with version 3.0.47301

When trying to call an endpoint that accepts a guid as part of the path I'm receiving this exception

```
System.ArgumentOutOfRangeException: Index must be within the bounds of the List. (Parameter 'index')
   at System.Collections.Generic.List`1.Insert(Int32 index, T item)
   at Microsoft.Repl.Input.InputManager.StartAsync(IShellState state, CancellationToken cancellationToken) in /_/src/Microsoft.Repl/Input/InputManager.cs:line 326
   at Microsoft.HttpRepl.Program.Start(String[] args, IConsoleManager consoleManager, IPreferences preferences) in /_/src/Microsoft.HttpRepl/Program.cs:line 75
   at Microsoft.HttpRepl.Program.Main(String[] args) in /_/src/Microsoft.HttpRepl/Program.cs:line 24
   at Microsoft.HttpRepl.Program.<Main>(String[] args)
```

Steps to reproduce:
- Connect to domain that supports swagger
- cd to a path that supports a GET with a Guid as a parameter eg: /somepath/{someguid}
- type `GET ` and paste in the guid
- hitting enter or space at this point results in the exception being raised

**Comment by tlmii** [comment]
This was fixed in #366
